doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
+source is ``$MNE_ROOT/bin/mne_setup``.  If you don't know what shell you are
+using, you can run the following command to find out:
 
 .. code-block:: console
 
     $ echo $SHELL
 
-If the output indicates a POSIX shell ``bash`` or ``sh`` then you should export
-variables and source the bash/sh setup file with three commands:
+To configure MNE-C automatically for ``bash`` or ``sh`` shells, add this to
+your ``.bashrc``::
 
-.. code-block:: console
-
-    $ export MNE_ROOT=<MNE>
-    $ export MATLAB_ROOT=<Matlab>
-    $ . $MNE_ROOT/bin/mne_setup_sh
+    export MNE_ROOT=<path_to_MNE>
+    export MATLAB_ROOT=<path_to_MATLAB>
+    . $MNE_ROOT/bin/mne_setup_sh
 
-where ``<MNE>`` is replaced by the absolute path to where you have installed
-the MNE software. This path will be the parent path to the ./bin and ./lib
-subdirecotries in the MNE distribution. Or the command
-"ls $MNE_ROOT/bin/mne_setup_sh" should succeed if you have set MNE_ROOT
-correctly.
+where ``<path_to_MNE>`` and ``<path_to_MATLAB>`` are replaced by the absolute
+paths to MNE-C and MATLAB, respectively.
 
-If you do not have a Matlab distribution, then leave MATLAB_ROOT undefined.
+To configure MNE-C automatically for ``zsh``, use the built-in ``emulate``
+command in your ``.zshrc`` file::
 
-For the bash compatible ``zsh`` use the builtin emulate command to source
-the bash/sh setup script (after exporting variables):
+    export MNE_ROOT=<path_to_MNE>
+    export MATLAB_ROOT=<path_to_MATLAB>
+    emulate sh -c 'source $MNE_ROOT/bin/mne_setup_sh'
 
-.. code-block:: console
+To configure MNE-C automatically for ``csh`` or ``tcsh`` shells, the
+corresponding commands in the ``.cshrc`` / ``.tcshrc`` file are::
 
-    $ export MNE_ROOT=<MNE>
-    $ export MATLAB_ROOT=<Matlab>
-    $ emulate sh -c 'source $MNE_ROOT/bin/mne_setup_sh'
+    setenv MNE_ROOT <path_to_MNE>
+    setenv MATLAB_ROOT <path_to_MATLAB>
+    source $MNE_ROOT/bin/mne_setup
 
-For ``csh`` or ``tcsh`` shell, the corresponding commands are:
+If you have done this correctly, the command ``ls $MNE_ROOT/bin/mne_setup_sh``
+should succeed when run in a new terminal.
 
-.. code-block:: console
+Testing your installation
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    $ setenv MNE_ROOT <MNE>
-    $ setenv MATLAB_ROOT <Matlab>
-    $ source $MNE_ROOT/bin/mne_setup
-
-
-For BEM mesh generation using the watershed algorithm or
-on the basis of multi-echo FLASH MRI data (see :ref:`create_bem_model`) and
-for accessing the tkmedit program
-from mne_analyze, see :ref:`CACCHCBF`,
-the MNE software needs access to a FreeSurfer license
-and software. Therefore, to use these features it is mandatory that
-you set up the FreeSurfer environment
-as described in the FreeSurfer documentation.
-
-The environment variables relevant to the MNE software are
-listed in :ref:`CIHDGFAA`.
-
-.. tabularcolumns:: |p{0.3\linewidth}|p{0.55\linewidth}|
-.. _CIHDGFAA:
-.. table:: Environment variables
```

Comment:
I ditched the table, since 4 of the 6 variables are discussed above, and the other two are discussed on the `mne_process_raw` page.